GovUkDesignSystem/GovUkDesignSystemComponents/SummaryListViewModel.cs

```diff
@@ -43,6 +43,8 @@ public class SummaryListRowViewModel
         ///     Add row action. 
         /// </summary>
         public SummaryListRowActionViewModel Actions { get; set; }
+
+        public bool AnyRowHasActions { get; set; }
```

Comment:
I think this would be nicer as `ShowActions` or similar

Reply:
Happy to change, but just want to confirm this as AnyRowHasActions is the name given to the property in the actual GovUk njk file https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/components/summary-list/template.njk

Reply:
In the GovUK code they are doing the whole table in one go, this property is specifically on a row object and it's an implementation detail rather than part of the API, so I think it should change

Reply:
Have done then :)